pkg/daemon/ceph/client/filesystem.go

```diff
@@ -231,7 +232,7 @@ func WaitForActiveRanks(
 	}
 	logger.Infof("waiting %.2f second(s) for number of active mds daemons for fs %s to become %s",
 		float64(timeout/time.Second), fsName, countText)
-	err := wait.Poll(3*time.Second, timeout, func() (bool, error) {
+	err := wait.PollUntilContextTimeout(ctx.TODO(), 3*time.Second, timeout, true, func(ctx ctx.Context) (bool, error) {
```

Comment:
Is clusterInfo's Context not available before this is called?

```suggestion
	err := wait.PollUntilContextTimeout(clusterInfo.Context, 3*time.Second, timeout, true, func(ctx ctx.Context) (bool, error) {
```